### ISSUE TITLE
NXtransformations is an invalid class in NXinstrument

### ISSUE THREAD

**Issue by jkotan** [nexpy/nxvalidate#17]
I get 
```
            NXtransformations: /scan/instrument/transformations
                NXtransformations is an invalid class in NXinstrument
```
which can be related to https://github.com/nexusformat/definitions/issues/1471

**Comment by rayosborn** [comment]
See the comment in #6. Most of these were fixed in [v0.3.0-beta.1](https://github.com/nexpy/nxvalidate/releases/tag/0.3.0-beta.1), but I need to add `NXgeometry` and `NXtransformations`. I will release that as v0.3.2-beta.1 later today.

**Comment by rayosborn** [comment]
This should be fixed in [v0.3.2.b1](https://github.com/nexpy/nxvalidate/releases/tag/0.3.2-beta.1).

**Comment by jkotan** [comment]
Thanks
